### DISCUSSION TITLE
Star goods

### DISCUSSION THREAD

**Discussion by No-Nothing** [Stellarium/stellarium#2204] [General]
Which hardware requirements apply to the individual switchable star catalogs? I have the impression that the hardware requirements accelerate exponentially, at least drastically, with each catalog. - How can I delete the catalogues if necessary? Every zoom through the galaxy overwhelms my computing time. Again, are there hardware recommendations for the star catalogs 1,2,3(ff).?

**Comment by gzotti** [comment]
What are your demands and expectations in computing speed (frame rate)?  Stellarium runs well on a €35 Raspberry Pi 3B+ on a 10" screen, but possibly for low expectations  of "well" (15fps without satellites). A Raspberry Pi 4 generally seems to work better than a 1GHz AMD dual-core netbook from 2015. It runs well on my 2008 CoreDuo 2 laptop, great on a Geforce 580GTX laptop from 2012, but of course still faster on a Geforce 3090 system.

There are several factors: Multicore CPU (I doubt more than 4 cores will speed it up significantly, though), 4GB or more of RAM (recommended), graphics card, screen resolution, many configurable options like number of minor planets/comets, number of satellites, sky brightness (light pollution and star size details) settings (which influence how many stars are actually read and displayed), and somewhere down the line also number of configured star catalogs. But having installed the most complete catalog should not slow down naked-eye view except for very old or very low-end systems. You just can zoom in with a more complete view.

**Comment by No-Nothing** [comment]
In any case, it has to do with the DSS & HPS buttons of the software and with the many GAIA catalogs, I mean to imagine. But how am I supposed to know that.

 

So what is the reason for the snail-like speed in the Fast Forward Star Planetarium?

 

It does not fail because of my hardware, the Win10 is quite still moderately suitable for gaming. My Mini PC could run Win11, the graphics is implemented in the CPU. Both computers offer similar performance. One on three times more capacity than the other. But a performance difference is hardly noticeable or observable.

 

The Stellarium configurations were not synchronised, they simply resulted when viewing the planetarium, depending on the situation. So viewed as found. Probably with different misalignments.

 

I didn't connect my own telescope, it stopped being interesting at the interface. It's just complicated what happens there, system barriers.

 

I only selected possible configurations, did not change any code.

 

Simulating astronomical situations is temptingly easy, even without prior knowledge.

 

 

I can't get any closer to solving the riddle via the hardware, it must have something to do with the use of the software, something switched on, too many functions activated or the computing power of the computers occupied with quibbles towards zero.

 

So again, what is most likely the reason for the snail-like zoom speed in the Fast Forward Stars Planetarium?

 

Von: Georg Zotti ***@***.***> 
Gesendet: Samstag, 22. Januar 2022 14:20
An: Stellarium/stellarium ***@***.***>
Cc: No-Nothing ***@***.***>; Author ***@***.***>
Betreff: Re: [Stellarium/stellarium] Star goods (Discussion #2204)

 

What are your demands and expectations in computing speed (frame rate)? Stellarium runs well on a €35 Raspberry Pi 3B+ on a 10" screen, but possibly for low expectations of "well" (15fps without satellites). A Raspberry Pi 4 generally seems to work better than a 1GHz AMD dual-core netbook from 2015. It runs well on my 2008 CoreDuo 2 laptop, great on a Geforce 580GTX laptop from 2012, but of course still faster on a Geforce 3090 system.

There are several factors: Multicore CPU (I doubt more than 4 cores will speed it up significantly, though), 4GB or more of RAM (recommended), graphics card, screen resolution, many configurable options like number of minor planets/comets, number of satellites, sky brightness (light pollution and star size details) settings (which influence how many stars are actually read and displayed), and somewhere down the line also number of configured star catalogs. But having installed the most complete catalog should not slow down naked-eye view except for very old or very low-end systems. You just can zoom in with a more complete view.

—
Reply to this email directly, view it on GitHub <https://github.com/Stellarium/stellarium/discussions/2204#discussioncomment-2022687> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AM4JKCPNN7IBW4ONPJRGG4DUXKVHXANCNFSM5MRFF4FA> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .

**Reply by gzotti** [reply]
I cannot comment on a Fast Forward Stars Planetarium, we discuss Stellarium for Desktop here. I wish we had GAIA catalogs already...

When you activate DSS or HiPS, these data are retrieved via Internet. Of course this slows things down. Just what do you expect?

Also on-screen labels are "expensive", the standard settings should be OK.

**Reply by No-Nothing** [reply]
Dear gzotti, you are so funny. That' s what i expected. But, there was still hope for any other information that upgrades my stellarium experience.  Think I'm a fool, I expected to get more recognition, if there is no virtual revolution for me, then at least some progress in terms of complicity among jugglers who leave no significant difference between planetariums and hebaria. Any questions, please do not hesitate to contact us. May there be sense.
